### PR TITLE
New GCChop class with tests.

### PR DESCRIPTION
**Pull request by vcristiani**
New GCChop class that implements the dynamic decomposition method that uses a cutoff value in circularity parameter.

### CODE REVIEW

**Comment by RafaelPignata622** [comment]
Everything looks good.

**Review by RafaelPignata622 — APPROVED** [review]

**Review by antotaverna — APPROVED** [review]
Is eps_cut=0.6 belong to some paper?

**Review by nvillagra — COMMENTED** [review]
Everything seems to be functioning properly

**Review by nvillagra — APPROVED** [review]

File: tests/test_models.py [hunk 1]
```diff
@@ -84,6 +84,60 @@ def test_GCAbadi_histogram(mock_real_galaxy):
...
+    gal = mock_real_galaxy
+    chop = models.GCChop()
+    chop.decompose(gal)
+    labels = chop.labels_
+
+    (comp0,) = np.where(labels == 0)
+    (comp1,) = np.where(labels == 1)
+
+    X, y = gal.values()
+    clean_eps = np.where(~np.isnan(X[:, 8]))[0]
```
Comment (ornemarioni) [inline tests/test_models.py:127]: Change this line for:  clean_eps = ~np.isnan(X[:, 8])

File: tests/test_models.py [hunk 2]
```diff
@@ -84,6 +84,60 @@ def test_GCAbadi_histogram(mock_real_galaxy):
...
+    expected_esf = np.where(X[clean_eps, 8] <= 0.6)[0]
```
Comment (ornemarioni) [inline tests/test_models.py:128]: Why don't you use tuples anymore?

**Review by ornemarioni — COMMENTED** [review]
Ojo con el array empty...

File: tests/test_models.py (hunk 2 above)
Comment (vcristiani) [reply tests/test_models.py:128]: I still use the tuples, np.where returns a tuple. This line stores whatever is in the first element of that tuple in the clean_eps or expected_esf variable.

**Pull request merged**